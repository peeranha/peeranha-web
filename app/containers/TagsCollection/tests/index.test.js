import { TagsCollection } from '../index';

describe('<TagsCollection />', () => {
  const props = {
    locale: 'en',
    profile: {},
    communities: [{ id: 1, tags: [] }],
    communitiesLoading: false,
    showLoginModalDispatch: jest.fn(),
  };

  it('render', () => {
    expect(TagsCollection(props)).toMatchSnapshot();
  });
});
