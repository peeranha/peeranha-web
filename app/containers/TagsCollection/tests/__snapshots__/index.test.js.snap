// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TagsCollection /> render 1`] = `
<div>
  <UNDEFINED
    keywords={
      Array [
        Array [],
      ]
    }
    language="en"
    title="Tags collection"
  />
  <UNDEFINED />
  <UNDEFINED
    communities={
      Array [
        Object {
          "id": 1,
          "tags": Array [],
        },
      ]
    }
  />
  <UNDEFINED />
</div>
`;
